py/strato/common/log/configurelogging.py

```diff
@@ -40,6 +40,13 @@ def configureLogging(name, forceDirectory=None, registerConfigurationReloadSigna
         hostname = subprocess.check_output('/bin/hostname').strip()
     if registerConfigurationReloadSignal:
         _configureLoggingSignalHandlers()
+    if os.environ.get("MONKEY_ELASTIFY"):
```

Comment:
where do you check it is Monkey who called the configureLogging?
i think not only monkey uses pycommonlog